### PR TITLE
add rpc for light client proof

### PR DESCRIPTION
**Pull request by bowenwang1996**
Add `experimental_lightClientProof` to provider to support querying light client proofs, which is needed for the Eth <> NEAR bridge to work. It is hard to write a test now because the rpc is not yet supported on testnet.

### CODE REVIEW

File: src/providers/json-rpc-provider.ts [hunk 1]
```diff
@@ -115,12 +115,20 @@ export class JsonRpcProvider extends Provider {
         return await this.sendJsonRpc('EXPERIMENTAL_genesis_config', []);
     }
 
+    /**
+     * Gets EXPERIMENTAL_light_client_proof from RPC
```
Comment (vgrichina) [inline src/providers/json-rpc-provider.ts:119]: Would be nice to provide a bit more info on what is this about (maybe link to relevant nearcore info?).

**Comment by vgrichina** [comment]
@bowenwang1996 check `startFromVersion` in `test/account.test.js` – should be possible to do the same for testing not released yet features

**Comment by bowenwang1996** [comment]
@vgrichina that would be true if we have proper versioning.

**Comment by vgrichina** [comment]
> @vgrichina that would be true if we have proper versioning.

Then we need to fix versioning in nearcore.  @chefsale we need to figure this out. It should be possible to gate tests to run only on newer version of nearcore and looks like according to @bowenwang1996 there is something missing on nearcore side of things.

**Comment by bowenwang1996** [comment]
@vgrichina yes it is blocked partially on the devops side.

**Pull request merged**